content/docs/start/model-management/model-cicd.md

```diff
@@ -0,0 +1,217 @@
+---
+title: 'Get Started: Using and deploying models'
+description:
+  'Easily download your models from the model registry. Set up your CICD
+  pipelines to be trigger by model registry actions (such as assigning model
+  stages) and deploy models directly form the model registry.'
+---
+
+# Get Started: Using and Deploying Models
+
+In this chapter, we will learn how to access and use models and how to use the
+model registry to trigger automated CICD model workflows.
+
+## Pushing model artifacts to the remote
+
+If you are following along from the
+[model registry chapter](/doc/start/model-management/model-registry), you have
+trained a model and registered it to the model registry. However, to start using
+and sharing the model artifacts we need to push them to our DVC remote.
+
+Provided that you have the
+[DVC remote set up](/doc/start/data-management/data-versioning#configuring-a-remote),
+you now just need to call
+
+```console
+dvc push
+```
+
+from our project's git repository. All data and artifacts (such as models) which
+we created and which are versioned by DVC will now be uploaded from our local
+DVC cache to the DVC Remote.
+
+<admon type="tip">
+
+To learn about how the DVC Remote works and how to set it up, have a look at the
+[Get Started guide on Data management](/doc/start/data-management/data-versioning)
+or the [Remote Storage Guide](/doc/user-guide/data-management/remote-storage)
+for even more detail.
+
+</admon>
+
+## Downloading models
+
+It is useful to download model artifacts for example for local testing or for
+use in CICD workflows. With models versioned by DVC this can be done easily by
+using the `dvc artifact get` CLI command as follows
+
+```bash
+dvc artifacts get https://<path-to-my-git-forge-account>/example-get-started-experiments.git pool-segmentation
```

Comment:
```suggestion
dvc artifacts get https://github.com/<user>/example-get-started-experiments.git pool-segmentation
```

Since we are building off the GH example, let's simplify this to only the user.